app/controllers/shipping_controller.rb

```diff
@@ -0,0 +1,47 @@
+require 'active_shipping'
+require_relative '../models/post_log.rb'
+
+class ShippingController < ApplicationController
+  USPS = ActiveShipping::USPS.new(login: ENV['USPS_USERNAME'])
+  UPS = ActiveShipping::UPS.new(login: ENV["UPS_LOGIN"], password: ENV["UPS_PASSWORD"], key: ENV["UPS_KEY"])
+
+  #Kept for testing purposes.
+  def index
+    render json: ["hello world"]
+  end
```

Comment:
This was smart, to make up a route for testing! Just make sure that in a professional setting you take these out before you PR 🤓